rotkehlchen/history/price.py

```diff
@@ -151,6 +152,12 @@ def get_price_for_special_asset(
                 timestamp=timestamp,
             )
             return Price(usd_price * price_mapping)
+        if from_asset == A_POLYGON_POS_MATIC and timestamp > POLYGON_POS_POL_HARDFORK:
+            return PriceHistorian().query_historical_price(
+                from_asset=Asset('eip155:1/erc20:0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6'),  # POL token  # noqa: E501,
```

Comment:
it's a static method

```suggestion
            return PriceHistorian.query_historical_price(
                from_asset=Asset('eip155:1/erc20:0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6'),  # POL token  # noqa: E501,
```